### PR TITLE
fix debugger tests with new EMITTING probe status

### PR DESCRIPTION
**Pull request by jpbempel**
## Motivation

Following https://github.com/DataDog/dd-trace-java/pull/6365 a new probe status was introduced, requiring to update those tests

## Workflow

1. ⚠️ Create your PR as draft ⚠️
2. Work on you PR until the CI passes (if something not related to your task is failing, you can ignore it)
3. Mark it as ready for review
    * Test logic is modified? -> Get a review from RFC owner. We're working on refining the `codeowners` file quickly.
    * Framework is modified, or non obvious usage of it -> get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)

:rocket: Once your PR is reviewed, you can merge it!

🛟 [#apm-shared-testing](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X) 🛟

## Reviewer checklist

* [ ] [Relevant label](https://github.com/DataDog/system-tests/blob/main/docs/CI/labels.md) (`run-parametric-scenario`, `run-profiling-scenario`...) are presents
* [ ] No system-tests internal is modified. Otherwise, I have the approval from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
* [ ] CI is green, or failing jobs are not related to this change (and you are 100% sure about this statement)
* [ ] A docker base image is modified?
    * [ ] the relevant `build-XXX-image` label is present
    * [ ] To R&P team: locally build and push the image to hub.docker.com 
* [ ] A scenario is added (or removed)?
    * [ ] Get a review from [R&P team](https://dd.enterprise.slack.com/archives/C025TJ4RZ8X)
    * [ ] Once merged, add (or remove) it in system-test-dasboard nightly

### CODE REVIEW

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
There are tests failures. Probably you should split the method "test_method_probe_snaphots" and use the decorators @missing_feature setting the version of the tracer that releases this feature.
Be careful, as it will impact in the test for other lang/tracers.

**Comment by jpbempel** [comment]
> There are tests failures. Probably you should split the method "test_method_probe_snaphots" and use the decorators @missing_feature setting the version of the tracer that releases this feature. Be careful, as it will impact in the test for other lang/tracers.

Thanks for the hint. Fixed

File: tests/debugger/test_debugger.py [hunk 1]
```diff
@@ -194,6 +194,7 @@ def setup_method_probe_snaphots(self):
         self.span_probe_response = weblog.get("/debugger/span")
         self.span_decoration_probe_response = weblog.get("/debugger/span-decoration/asd/1")
 
+    @missing_feature(context.library >= "java@1.27", reason="introduction of new EMITTING probe status")
```
Comment (robertomonteromiguel) [inline tests/debugger/test_debugger.py:197]: But with the latest changes we aren't testing the "EMITTING" status....
I guess it should be something  similar to this:

```
def method_probe_snaphots(self,  expected_probes):
    ... assertions....
@irrelevant(context.library >= "java@1.27", reason="introduction of new EMITTING probe status")
def test_method_installed_probe_snaphots(self):
        expected_probes = {
            "log170aa-acda-4453-9111-1478a6method": "INSTALLED",
            "metricaa-acda-4453-9111-1478a6method": "INSTALLED",
            "span70aa-acda-4453-9111-1478a6method": "INSTALLED",
            "decor0aa-acda-4453-9111-1478a6method": "INSTALLED",
        }
       self.method_probe_snaphots(expected_probes)

@missing_feature(context.library < "java@1.27", reason="last releases use new EMITTING probe status")
def test_method_emitting_probe_snaphots(self):
        expected_probes = {
            "log170aa-acda-4453-9111-1478a6method": "EMITTING",
            "metricaa-acda-4453-9111-1478a6method": "EMITTING",
            "span70aa-acda-4453-9111-1478a6method": "EMITTING",
            "decor0aa-acda-4453-9111-1478a6method": "EMITTING",
        }
       self.method_probe_snaphots(expected_probes)
```

**Review by robertomonteromiguel — CHANGES_REQUESTED** [review]
with the latest changes we aren't testing the "EMITTING" status....

**Review by robertomonteromiguel — APPROVED** [review]

**Pull request merged**